### PR TITLE
Adjusting Raider Ghost Role

### PR DESCRIPTION
**Pull request by Tron-117**
I'm seeing the value in adding the raider ghost role as a sort of antag to spice things up. I must see how players interact with this role to know how to move forward with it.

## About The Pull Request
Will be doing some experiments in the coming days to see how players use the roles/react to them.
Also changed some of the damage coefficient on the priest CoR mobs. I decided to put it in this PR as it's not really a big change.

## Why It's Good For The Game
Ghost roles may actually bring some value to the game, might as well make them somewhat wortwhile.

## Pre-Merge Checklist
- [x] Your Pull Request contains no breaking changes
- [x] You tested your changes locally, and they work.
- [x] There are no new Runtimes that appear.
- [x] You documented all of your changes.

## Changelog
:cl:
tweak: Added two extra grease gun mags, two stims, and a headset to the raider ghost role
tweak: changes some of the damage coefficient values on two CoR mobs
tweak: Adds a couple traits to a halloween event species
/:cl:

### CODE REVIEW

**Review by zuliouse — COMMENTED** [review]
Conflicts with https://github.com/FortunaSS13/Fortuna/pull/413

**Comment by buffyuwu** [comment]
let me know when you'd like a review

**Comment by Tron-117** [comment]
> let me know when you'd like a review

Review please (I could've sworn there was a request review button on this page)

File: code/game/objects/structures/ghost_role_spawners.dm [hunk 1]
```diff
@@ -837,12 +837,21 @@
 	uniform = /obj/item/clothing/under/f13/ravenharness
 	suit = /obj/item/clothing/suit/armor/f13/combat/mk2/raider
 	head = /obj/item/clothing/head/helmet/f13/combat/mk2/raider
+	ears = /obj/item/radio/headset
 	belt = /obj/item/storage/belt/military/assault
 	shoes = /obj/item/clothing/shoes/jackboots
 	back = /obj/item/storage/backpack/satchel/leather
 	backpack_contents = list(
-		/obj/item/ammo_box/magazine/greasegun = 2,
+		/obj/item/ammo_box/magazine/greasegun = 4,
```
Comment (buffyuwu) [inline code/game/objects/structures/ghost_role_spawners.dm:845]: we will want this to remain 2 to match the other ghost roles
```suggestion
		/obj/item/ammo_box/magazine/greasegun = 2,
```

**Review by buffyuwu — CHANGES_REQUESTED** [review]

**Review by buffyuwu — APPROVED** [review]

**Pull request merged**